apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -268,6 +265,28 @@ describe('The JSDebugger component', () => {
   });
 });
 
+function moveMouse(xTarget, yTarget) {
```

Comment:
Worth de-duplicating this code against `VisualizationResizeBarTest.jsx`? I could see the shared code going in a `test/util/...` file.

https://github.com/code-dot-org/code-dot-org/blob/2a2c4207ac702abaa90af0eb6640a80d4edd3462/apps/test/unit/lib/ui/VisualizationResizeBarTest.jsx#L65-L85